deepmd/pt/model/task/denoise.py

```
@@ -1,79 +1,300 @@
+class DenoiseFittingNet(Fitting):
     def __init__(
         self,
-        feature_dim,
-        ntypes,
-        attn_head=8,
-        prefactor=[0.5, 0.5],
-        activation_function="gelu",
+        ntypes: int,
+        dim_descrpt: int,
+        embedding_width: int,
+        neuron: list[int] = [128, 128, 128],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)